service/statistic.js

```diff
@@ -0,0 +1,80 @@
+import { UnexpectedError } from '../validators/errors';
+
+import userRepository from '../repository/users';
+import equationRepository from '../repository/equation';
+import expressionRepository from '../repository/expression';
+import { ADMIN_ROLE } from '../constants';
+
+class StatisticService {
+  async getUserRoleStatistic() {
+    try {
+      const queryResult = await userRepository.getUserRoleStatistic();
+
+      var data = {
+        admin: 0,
+        users: 0,
+      };
+
+      queryResult.forEach(({ dataValues: element }) => {
```

Comment:
How many times this loop will be executed? You have only two roles and since you group by role in the query you will get only two rows. I'm not saying this is wrong but it is adding useless complexity and i think that your `queryResult.dataValues` is holding the same object you are creating here.

Reply:
DataValues is an array of objects so this is how I need to transform it to get the final result.